EXAMPLES.md

```diff
@@ -160,3 +161,36 @@ $tbxFile = $crowdin->glossary->download($glossaryId, $glossaryExport->getIdentif
 
 print_r($tbxFile->getData());
 ```
+
+## Manage file references
```

Comment:
File references are not something so common, there is no need in the dedicated example